pkg/agent/agent.go

```diff
@@ -198,7 +200,32 @@ func (a *Agent) setDefaultConfig() error {
 		},
 	}
 
-	return config.Save()
+	err = config.Save()
+	if err != nil {
+		return err
+	}
+
+	//	as the agent is running as root, we need to move the config to the user's home directory
```

Comment:
Why is this necessary?

Reply:
Okay, I see why but I think a symlink might be more appropriate since this would prevent the root user (which runs the agent) from using daytona commands.

Reply:
root user has no need to run the daytona command

Reply:
Okay